frontend/public/components/search.tsx

```diff
@@ -147,13 +142,14 @@ const SearchPage_: React.FC<SearchProps & StateProps & DispatchProps> = (props)
   const pinToggle = (e: React.MouseEvent<HTMLElement>, resource: string) => {
     e.preventDefault();
     e.stopPropagation();
-
-    const index = props.pinnedResources.indexOf(resource);
-    if (index >= 0) {
-      confirmNavUnpinModal(resource, pinnedResources, props.onPinnedResourcesChange);
-      return;
+    if (pinnedResourcesLoaded) {
```

Comment:
Unnecessary because `pinToggle` is only available to be called when `pinnedResourcesLoaded === true`

Reply:
@sahil143  ^^